docs/RBAC.md

```diff
@@ -94,14 +94,13 @@ implementation has drifted as we learned the limitations, this change realigns s
 Find the managed clusters that the user is authorized to view and [cache](#cache) the results.
 > 1. Get all the namespaces associated with a managed cluster. We do this once for all users.
 >       - CLI: `oc get ManagedClusters`
-> 2. Get all projects (namespaces) for the user. (We already have this data.)
->       - CLI: `oc projects --as=<user>`
+> 2. Get all namespaces. We already have this data (Step 3 in Hub Cluster)
+>       - CLI: `oc get namespaces`
 >       - API: [ProjectList](https://docs.okd.io/3.9/rest_api/apis-project.openshift.io/v1.Project.html#Get-apis-project.openshift.io-v1-projects)
```

Comment:
We don't need to get all namespaces, we only need the namespaces associated with ManagedClusters (step 1).
```suggestion
```